torch/quantization/fx/qconfig_utils.py

```diff
@@ -86,6 +86,27 @@ def get_module_name_regex_qconfig(qconfig_dict, module_name, fallback_qconfig):
     return fallback_qconfig
 
 
+def maybe_adjust_qconfig_for_module_name_function_order(
```

Comment:
should we merge this function with get_object_type_qconfig to make the precedence more clear?

Reply:
yes, I looked into this, it's not clean with the current code.  We might need to do some refactoring of the qconfig utils to make this easier, in a future PR.